### PR TITLE
[wip] feat: add static openrpc.json file 

### PR DESCRIPTION
**Pull request by meowsbits**
> see https://github.com/etclabscore/ECLIPs/pull/3 for details
@BelfordZ 

Copied from https://github.com/etclabscore/ethereum.go-ethereum/pull/6

### CODE REVIEW

**Comment by BelfordZ** [comment]
What is needed to get this in @meowsbits ?

**Comment by meowsbits** [comment]
Handing question to @r8d8

File: openrpc.json [hunk 1]
```diff
@@ -0,0 +1,1595 @@
...
+      "name": "etclabs core"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "servers": [
+    {
+      "url": "http://localhost:8080"
```
Comment (shanejonas) [inline openrpc.json]:
```suggestion
      "url": "http://localhost:8545"
```

File: openrpc.json [hunk 2]
```diff
@@ -0,0 +1,1595 @@
+{
+  "openrpc": "1.0.0-rc1",
+  "info": {
+    "version": "1.0.0",
+    "title": "Ether RPC",
```
Comment (shanejonas) [inline openrpc.json]:
```suggestion
    "title": "Multi Geth RPC",
```

File: openrpc.json [hunk 3]
```diff
@@ -0,0 +1,1595 @@
+{
+  "openrpc": "1.0.0-rc1",
+  "info": {
+    "version": "1.0.0",
+    "title": "Multi Geth RPC",
+    "description": "The etclabs brings you... generated RPC!",
```
Comment (shanejonas) [inline openrpc.json]:
```suggestion
    "description": "This API lets you interact with the blockchain via JSON-RPC",
```

**Review by BelfordZ — APPROVED** [review]

**Pull request closed without merging**